### DISCUSSION TITLE
Projects (beta) - "group by" hide field which is grouped by

### DISCUSSION THREAD

**Discussion by a-a-hofmann** [community/community#5512] [Projects and Issues] [answered]
It would be nice if, when grouping by field, there were an option to hide that field's column since the information is already present as a header.

This would help in reducing visual clutter when you have many columns.

![image](https://user-images.githubusercontent.com/10435742/132362161-bbdc6c98-7298-462d-8763-a079b0753506.png)

**Comment by zepfietje** [comment]
Just started using the new projects, and this is the first thing that I ran into.
Would be nice to have an option to hide the field which is grouped by to reduce duplicate information.

**Comment by CodeCox** [comment]
And  _drag&drop_ can still be used to move a row  to a new desired location - which effectively will update this hidden field  so the edit/modify functionality will not be lost even if may be slightly compromised.

**Comment by zepfietje** [comment, marked as answer]
It seems like this has been implemented! 🎉

**Reply by a-a-hofmann** [reply]
Yep just noticed this as well! 🎉 
I'm going to go ahead and mark this as answered 😄
